packages/tpetra/core/src/Tpetra_Import_Util2.hpp

```diff
@@ -1311,7 +1311,10 @@ lowCommunicationMakeColMapAndReindex (
         const int PID = owningPIDs_view[j];
         auto outcome = RemoteGIDs_view_map.insert(GID, PID);
         if(outcome.success() && PID == -1) {
+// FIXME_SYCL Enable again once a SYCL conforming printf implementation is available.
+#ifndef KOKKOS_ENABLE_SYCL
           printf("Cannot figure out if ID is owned.\n");
+#endif
```

Comment:
Why not just
```suggestion
```
?

Reply:
Fair question, I assumed the printf was a useful diagnostic in the package. @brian-kelley @csiefer2 would you prefer that I delete the printf altogether?

Reply:
Fine with me to delete it, especially since the ``Kokkos::abort`` message is exactly the same.